### PR TITLE
Migrate to attributes

### PR DESCRIPTION
**Pull request by tetienne**
Since several releases now, in order to ease code and number of lines, various `_attr_*` are available.

See [here](https://github.com/home-assistant/core/blob/fe17f97543b99f88993ebcb7fa15be0dfe5c43c7/homeassistant/components/climate/__init__.py#L181) for climate
 and [here](https://github.com/home-assistant/core/blob/fe17f97543b99f88993ebcb7fa15be0dfe5c43c7/homeassistant/helpers/entity.py#L269) for the common ones

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (tetienne) [inline .vscode/settings.json:1]: I forgot last time to add the VSCode configuration which allow to apply linter by default and launch HA through a task.

File: custom_components/toshiba_ac/entity.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+from __future__ import annotations
+
+from toshiba_ac.device import ToshibaAcDevice
+
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+from .const import DOMAIN
+
+
+class ToshibaAcEntity(Entity):
```
Comment (tetienne) [inline custom_components/toshiba_ac/entity.py:12]: Avoid to repeat yourself.

File: custom_components/toshiba_ac/climate.py [hunk 3]
```diff
@@ -349,16 +273,15 @@ def hvac_action(self):
 
         if self._device.ac_mode == ToshibaAcMode.AUTO:
             return CURRENT_HVAC_COOL  # CURRENT_HVAC_IDLE
-        elif self._device.ac_mode == ToshibaAcMode.COOL:
+        if self._device.ac_mode == ToshibaAcMode.COOL:
```
Comment (tetienne) [inline custom_components/toshiba_ac/climate.py:276]: As we return in the previous `if` block, `elif` is useless, use `if` instead.

File: custom_components/toshiba_ac/sensor.py [hunk 4]
```diff
@@ -70,18 +71,26 @@ async def async_setup_entry(hass, config_entry, async_add_devices):
...
+        super().__init__(toshiba_device)
+
+        self._ac_energy_consumption: ToshibaAcDeviceEnergyConsumption = None
+
+        self._attr_unique_id = f"{self._device.ac_unique_id}_sensor"
+        self._attr_name = f"{self._device.name} Power Consumption"
+        self.available = (
+            self._device.ac_id
+            and self._device.amqp_api.sas_token
+            and self._device.http_api.access_token
```
Comment (tetienne) [inline custom_components/toshiba_ac/sensor.py:92]: These tokens can be empty?

Comment (h4de5) [reply custom_components/toshiba_ac/sensor.py:92]: i think those should only be empty if authentification did not work

File: .vscode/settings.json (hunk 1 above)
Comment (h4de5) [inline .vscode/settings.json:1]: is there a way to have this file in git and have individial settings for this project that I do not want to have in git? like locale paths as I do not use the devcontainer.

Comment (tetienne) [reply .vscode/settings.json:1]: Sure, this configuration can be clean to remove some too opiniated settings. 
On side note, I highly recommand you to try the devcontainer, you will be able to easily test your custom component without any symlink or copy/paste, and [debug](https://www.home-assistant.io/integrations/debugpy/).

Comment (h4de5) [reply .vscode/settings.json:1]: i know about the devcontainer. For this project I do not have docker or the project locally. I access the docker daemon remotely using DOCKER_HOST and connect to the source using ssh.
sadly, I could not find a way to use a devcontainer when working remotely.

**Pull request merged**